src/Controls/src/Xaml/XmlTypeXamlExtensions.cs

```diff
@@ -68,8 +68,9 @@ static class XmlTypeXamlExtensions
 			for (var i = 0; i < lookupNames.Count; i++)
 			{
 				var name = lookupNames[i];
-				if (name.IndexOf(":", StringComparison.Ordinal) != -1)
-					name = name.Substring(name.LastIndexOf(':') + 1);
+				var lastIndex = name.LastIndexOf(':');
```

Comment:
This needs to specify the StringComparison. (The original version should have specified it, not sure how that got overlooked.)

Reply:
@hartez As MartyIX already stated, there is no such method for search by char, only searching by string.
FYI The LastIndexOf(char) method uses internally https://github.com/dotnet/runtime/blob/0b47a4dc519ada9c56d6f4d62865c2d96a56510c/src/libraries/System.Private.CoreLib/src/System/SpanHelpers.T.cs#L2223 which is in my opinion like ordinal compare.
The LastIndexOf(string, StringComparision) uses depending on nls settings from 
 here https://github.com/dotnet/dotnet/blob/8d5c52c5dd54417a2071c196663bb29ab1ee2442/src/runtime/src/libraries/System.Private.CoreLib/src/System/Globalization/Ordinal.cs#L443 to https://github.com/dotnet/dotnet/blob/b8bc661a7429baa89070f3bee636b7fbc1309489/src/runtime/src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs#L352.

Reply:
My mistake, carry on :)